src/index.ts

```diff
@@ -37,7 +37,11 @@ process.on("SIGINT", async () => {
         commandOptions: "",
     });
 
-    await deleteFolder(temporaryFolder);
+    try {
+        await deleteFolder(temporaryFolder);
+    } catch (error) {
+        logError(error as Error);
```

Comment:
I am not sure but I think you should convert this to a UserError, else it will show up in our Sentry.

Reply:
It won't show up in sentry because the error is not thrown anymore. It is just logged

Reply:
I was thinking the same thing but the problem is  it is not a UserError because it happens when the excution is inturupted midway, the User doesn't need to do anything to fix this error. I was thinking that on any other framework, if you stop the execution mid way and you get one such error you get the full stack. The other option would be to not log anything. If you think that's a better option i will change it

Reply:
@stefandarius the sentry error will still appear because that function sends an event when the error has type Error.

Given the fact that this code is a process cleanup and the user can't influence it, I would consider silencing it (if it fails don't show anything).

@Virgil993 what do you think?

Reply:
You're right. I wrote that function, but at a quick glance I thought it was a regular `log.error`.
I vote for silencing it too.

Reply:
But i still think we should show a debug log